### PR TITLE
correct susie part, xi, & elbo 

### PR DESCRIPTION
**Pull request by hkim89**
Need to add initial value functions for glm.

### CODE REVIEW

File: src/VEM.jl [hunk 1]
```diff
@@ -91,7 +90,8 @@ yt::Vector{Float64},Xt::Matrix{Float64},Xt₀::Matrix{Float64},β::Vector{Float6
             B1[:,1] = Diagonal(Sig1[:,1])*getXy('T',Xt,Z0) 
            # compute α_1
             Z =  0.5*(getXy('T',Xt,Z0)./sqrt.(ϕ)).^2
-            A0[:,1] = Π.*exp.(Z./(1.0.+ϕ/σ0[1]))./sqrt.(σ0[1]*ϕ.+1)
+            A0[:,1] = Π.*exp.(Z./(1.0.+ϕ/σ0[1]))./sqrt.(ϕ.^(-1).+1)
```
Comment (gaow) [inline src/VEM.jl:93]: we should work on the log scale first, then exp() it. eg instead of A/B, we do exp(log(A) - log(B))

File: src/VEM.jl [hunk 2]
```diff
@@ -100,7 +100,8 @@ yt::Vector{Float64},Xt::Matrix{Float64},Xt₀::Matrix{Float64},β::Vector{Float6
             Z0= yt - λ.*(getXy('N',Xt₀,β) + getXy('N',Xt,sum(hcat(AB1[:,1:l-1],AB0[:,l+1:end]),dims=2)[:,1])+ghat)
             B1[:,l] = Diagonal(Sig1[:,l])*getXy('T',Xt,Z0)
             Z =  0.5*(getXy('T',Xt,Z0)./sqrt.(ϕ)).^2
-            A0[:,l] = Π.*exp.(Z./(1.0.+ϕ/σ0[l]))./sqrt.(σ0[l]*ϕ.+1)
+            A0[:,l] = Π.*exp.(Z./(1.0.+ϕ/σ0[l]))./sqrt.(ϕ.^(-1).+1)
```
Comment (gaow) [inline src/VEM.jl:103]: same comment as before. We should always avoid doing exp like this

**Comment by gaow** [comment]
For the record, this patch already fixed the domain error initially brought up in #11

**Comment by hkim89** [comment]
Possibly based on glm but not done yet since this is from glmm.

**Pull request merged**